src/components/Review/Applications/ApplicationsReviewTableSchema.js

```diff
@@ -6,7 +6,9 @@ export const columns = {
 };
 
 export const ApplicationStateLabel = Object.freeze({
+    UNVERIFIED: "Unverified",
```

Comment:
Do we want Unverified aaplications to show uo in the admin panel? I think it shouldn't

- Still not a valid application
- If we are victims of spam, they will show up

Reply:
cc @BrunoRosendo @Naapperas For more opinions

Reply:
What's the flow here? We can only accept them once they've confirmed their emails? Imo that's too much friction for a website where it's already hard to bring companies into. They need to register, confirm email, wait for the application, and finish registration before starting to create offers.

What if, instead of that, we let the companies use the website without confirming email (just like today), and use the `isVerified` prop like a Twitter verification? The company could have a warning (snack bar) after logging in and the students (and admins) would be able to know if the company is fully trust worthy.

Reply:
If i am not mistaken, in a reunion it was discussed that Unverified should be a state like PENDING APROVED and REJECTED , so you can filter your application to show unverified application or others. About the spam, as it stands only one application with the same email can exist , however that way we are still vonorable to be spamed with applications with different emails, one solution to that would be deleting all application that are unverified after 10 minutes (when you can longer validate them) , but of course that way having an unverified state would no longer make sense.

As it stands when creating an application your application will be Unverified and cannot be reviewed, you will than receive an email to verifie it, if you do so then your application become PENDING and is ready to be reviewed. 

The idea of being like twitter is an interesting ideia, however it may be better to discuss it with the rest of the team as it is a concept completely different from the original ideia( it may be better to discussed it on the next reunion). In that case isVerified should probably be on the company instead of the application and the way companies verifie their companies should be diferent and only done after they are aproved. The email with a link that expires in 10 minutes which is sent when the application is created would no longer make a lot of sense in that case, maybe we could have a button in the company profile where you could validate your company, that way we could still use the email with the link to validate it

Reply:
Yes, I think we should discuss this again. I know it wasn't the original idea but I was re-thinking about it myself

Reply:
> What's the flow here? We can only accept them once they've confirmed their emails? Imo that's too much friction for a website where it's already hard to bring companies into. They need to register, confirm email, wait for the application, and finish registration before starting to create offers.
> 
> What if, instead of that, we let the companies use the website without confirming email (just like today), and use the `isVerified` prop like a Twitter verification? The company could have a warning (snack bar) after logging in and the students (and admins) would be able to know if the company is fully trust worthy.

I really like the Twitter verification idea but there is something that is bothering me. I'm sorry if what I'm about to say is completely wrong, but wouldn't the spam issue persist in the offers page, as each account can potentially create up to five concurrent offers? Wouldn't the offer page or even our database become way too bloated?
An idea that i had was to develop a scheduled function that would run every hour or so (not sure of the appropriate time interval) that would erase all expired applications.

Reply:
I like the Twitter thing, in addition to said pop-up/snackbar, perhaps we could/should restrict some actions made by unverified companies, but this may cause more harm than good. It should be discussed in the weekly meting but I personally am leaning towards what @BrunoRosendo suggested.